CHANGELOG.md

```diff
@@ -8,10 +8,14 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [1.7.0] - 2024-10-29
+
 ### Added
 * Documenting OpenFGA release process [#1923](https://github.com/openfga/openfga/pull/1923)
 * Cache Controller to extend Sub-problems and Iterators lifetime in cache [#2006](https://github.com/openfga/openfga/pull/2006)
-* Add access control experimental feature [#1913](https://github.com/openfga/openfga/pull/1913)
+* Add an experimental access control feature [#1913](https://github.com/openfga/openfga/pull/1913)
+  Learn more about this feature and how to enable it [here](https://openfga.dev/docs/getting-started/setup-openfga/access-control)
+  If you do try it out, please provide feedback in the [GitHub Discussion](https://github.com/orgs/openfga/discussions/378)
```

Comment:
Haven't created the discussion yet, will open it once we're set release

Reply:
@rhamzeh will that link continue being correct when you create the discussion? right now it points to takes you to https://github.com/openfga/community/pull/378

Reply:
Pointing to the full discussions for now and we can update after release